gui/src/components/Taipy/Chart.tsx

```diff
@@ -118,6 +118,38 @@ const getValueFromCol = (values: TraceValueType | undefined, col: string): (stri
     return [];
 };
 
+const getDecimatorPayload = (
+    decimator: string | undefined,
+    plotRef: React.RefObject<HTMLDivElement>,
+    modes: string[],
+    columns: Record<string, ColumnDesc>,
+    traces: string[][],
+    relayoutData: PlotRelayoutEvent | undefined = undefined
```

Comment:
could use the form relayoutData?: PlotRelayoutEvent :-)
if ommitted, the value is undefined